.github/workflows/cd.yml

```
@@ -74,12 +69,18 @@ jobs:
           script: |
             set -e
             
+            echo "${{ secrets.FIREBASE_JSON_B64 }}" | base64 --decode > /home/ubuntu/serviceAccountKey.json
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)